### ISSUE TITLE
Update cdn-deploy-main.yml to be triggered on both main and 2.x

### ISSUE THREAD

**Issue by trackleft** [az-digital/arizona-bootstrap#605]
- [x] Rename cdn-deploy-main.yml to cdn-deploy-head.yml

- [x] Change name of workflows from Build & deploy CDN assets to Build & deploy CDN assets from HEAD

- [x] Change buckets from 
```         
 aws s3 sync --cache-control max-age=691200 dist/ s3://${{ secrets.AZ_DIGITAL_CDN_BUCKET }}/lib/arizona-bootstrap/main/
          aws cloudfront create-invalidation --distribution-id ${{ secrets.AZ_DIGITAL_CDN }} --paths /lib/arizona-bootstrap/main/*
```

https://github.com/az-digital/arizona-bootstrap/blob/2.x/.github/workflows/cdn-deploy-main.yml#L25-L26

```
          aws s3 sync --cache-control max-age=691200 dist/ s3://${{ secrets.AZ_DIGITAL_CDN_BUCKET }}/lib/arizona-bootstrap/${GITHUB_REF}/
          aws cloudfront create-invalidation --distribution-id ${{ secrets.AZ_DIGITAL_CDN }} --paths /lib/arizona-bootstrap/${GITHUB_REF}/*
```

- [ ] Replace references to main bucket in docs to `2.x` on the `2.x` branch.

**Comment by trackleft** [comment]
Blocked by our first 5.x PR being merged into main.

**Comment by trackleft** [comment]
Need to switch which release action to trigger based on which branch has a release (from repository dispatch) since it is likely that different steps will be required for 5.x releases and 2.x releases.
